nextflow_schema.json

```
@@ -258,10 +258,11 @@
                 },
                 "ascat_ploidy": {
                     "type": "string",
```

Why is this a string?
```suggestion
                    "type": "number",
```